stdlib/public/Quote/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+#===-------------- CMakeLists.txt - Build the Quote library ---------------===#
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+#===----------------------------------------------------------------------===#
+#
+# SWIFT_ENABLE_TENSORFLOW
+#
+#===----------------------------------------------------------------------===#
+
+if(NOT SWIFT_ENABLE_TENSORFLOW)
+  return()
+endif()
+
+message(STATUS "Building Quote.")
+
+set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+set(swift_stdlib_compile_flags "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}")
+list(APPEND swift_stdlib_compile_flags "-DCOMPILING_QUOTE_MODULE")
+
+set(SOURCES "")
+
+# Copy Quote sources, if they exist.
+if (TENSORFLOW_SWIFT_QUOTE)
+  file(GLOB_RECURSE TENSORFLOW_SWIFT_QUOTE_SOURCES
+    "${TENSORFLOW_SWIFT_QUOTE}/Sources/Quote/*.swift")
```

Comment:
This is really an anti-pattern for CMake.  If you do this, subsequent incremental builds will be incorrect as this will not pick up new files being added - it is evaluated once.  It is much better to explicitly list out the sources.

Reply:
What's the right way of doing this? My knowledge of CMake is cursory at best, so I'd appreciate any help :)